.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -415,6 +415,26 @@ def instantiate_configs(only_slow_gradcheck):
             )
             c.dependent_tests.append(bc_breaking_check)
 
+        if (
+            compiler_name != "clang"
+            and not is_libtorch
+            and not is_vulkan
+            and not is_pure_torch
+            and not is_noarch
+            and not is_slow_gradcheck
+            and not only_slow_gradcheck
+        ):
```

Comment:
```suggestion
        if compiler_name != "clang"
           and not any(
               is_libtorch,
               is_vulkan
               is_pure_torch
               is_noarch
               is_slow_gradcheck
               only_slow_gradcheck
        ):
```

nit to make it a bit more readable (although this isn't really that much more readable admittedly)

Reply:
yeah it isn't that much more readable, how about let's just follow all other if statement format for now